ddpui/ddpairbyte/schema.py

```
@@ -1,8 +1,229 @@
-from typing import Optional
-from ninja import Schema
-from ddpui.ddpprefect.schema import DeploymentCurrentQueueTime
+class AirbyteJob(Schema):
```

Ruff (0.12.2)

Undefined name `Schema`

(F821)